### DISCUSSION TITLE
Video player is broken

### DISCUSSION THREAD

**Discussion by Ekrol34** [FreeTubeApp/FreeTube#5461] [General]
https://youtu.be/W-OMfccc3mc

**Comment by katero01** [comment]
I dont know if this is related but as of today if I use the local API as default the video will not play and either gives an error "DASH request error at 
URL: https://rr2---sn-jxou0gtapo3-2ime.googlevideo.com/videoplayback?expire" 
or it loops the thumbnail really fast. by changing the api to invidious it was able to work.

**Reply by Ekrol34** [reply]
Changed it to invidious, works fine!

**Reply by Dumaiden** [reply]
I swapped to the Invidious API and it sorted itself out. thank you

**Comment by user-7072** [comment]
The same is happening with me.What could be?

**Comment by rustyechelle** [comment]
new breaking changes from youtube i guess

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/5456 and see [workaround](https://github.com/FreeTubeApp/FreeTube/issues/5456#issuecomment-2248897869)

**Comment by ghost** [comment]
Idk if this is the same case but since like 2 hours ago playing a video results in a loop where this ordeal happens

NotSupportedError: Failed to load because no supported source was found.

https://github.com/user-attachments/assets/9c1b3320-88e9-4deb-af6e-037623b52c1e

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Its the same
